crates/nu-command/src/viewers/table.rs

```diff
@@ -1799,19 +1799,7 @@ enum TableView {
 }
 
 fn strip_output_color(output: Option<String>, config: &Config) -> Option<String> {
-    match output {
-        Some(output) => {
-            // the atty is for when people do ls from vim, there should be no coloring there
-            if !config.use_ansi_coloring || !atty::is(atty::Stream::Stdout) {
-                // Draw the table without ansi colors
-                Some(nu_utils::strip_ansi_string_likely(output))
-            } else {
-                // Draw the table with ansi colors
-                Some(output)
-            }
-        }
-        None => None,
-    }
```

Comment:
What are you smoking clippy?!

Reply:
That's crazy talk. I added this to my rust bump pr
```
#[allow(clippy::manual_filter)]
```